bridgehead

```diff
@@ -64,7 +64,7 @@ loadVars() {
 	fetchVarsFromVaultByFile /etc/bridgehead/$PROJECT.conf || fail_and_report 1 "Unable to fetchVarsFromVaultByFile"
 	setHostname
 	optimizeBlazeMemoryUsage
-	[ -e ./$PROJECT/vars ] && source ./$PROJECT/vars
+	[ -e ./$PROJECT/vars ] && source ./versions ./$PROJECT/vars
```

Comment:
Sorry to bother you again I meant sourcing `./versions` regardless of `./$PROJECT/vars` existing. I think it always exits so it should not really matter but if we have the check that anyway